src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const basePrice = PRICE_PER_DAY * days;
+
+  if (days >= SHORT_TERM && days !== LONG_TERM) {
+    return basePrice - SHORT_TERM_DISCOUNT;
```

Comment:
The condition `days >= SHORT_TERM && days !== LONG_TERM` is incorrect for applying the short-term discount. This condition will apply the short-term discount to any rental period greater than or equal to the short-term threshold, except when it equals the long-term threshold. Consider changing this condition to `days >= SHORT_TERM && days < LONG_TERM` to ensure the short-term discount is applied only to rentals that are at least the short-term threshold but less than the long-term threshold.